acm/src/main/java/org/literacybridge/acm/core/DataRequestService.java

```diff
@@ -1,114 +1,55 @@
 package org.literacybridge.acm.core;
 
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Collection;
 import java.util.List;
 import java.util.Locale;
-import java.util.Map;
 
 import org.literacybridge.acm.api.IDataRequestResult;
 import org.literacybridge.acm.api.IDataRequestService;
-import org.literacybridge.acm.categories.Taxonomy;
 import org.literacybridge.acm.config.ACMConfiguration;
-import org.literacybridge.acm.config.DBConfiguration;
-import org.literacybridge.acm.db.PersistentAudioItem;
-import org.literacybridge.acm.db.PersistentCategory;
-import org.literacybridge.acm.db.PersistentLocale;
-import org.literacybridge.acm.db.PersistentTag;
-import org.literacybridge.acm.gui.Application;
-import org.literacybridge.acm.index.AudioItemIndex;
+import org.literacybridge.acm.store.Category;
+import org.literacybridge.acm.store.MetadataStore;
+import org.literacybridge.acm.store.Playlist;
 
 public class DataRequestService implements IDataRequestService {
```

Comment:
As with DataRequestResult, this class should probably describe what it (concretely) is.